makefile

```diff
@@ -29,6 +29,14 @@ ifndef INSTALL_DIR
 	INSTALL_DIR=/usr
 endif
 
+ifndef INCLUDE_PATH
+	INCLUDE_PATH=${INSTALL_DIR}/include
+endif
+
+ifndef LIBRARY_PATH
+	LIBRARY_PATH=${INSTALL_DIR}/lib
+endif
+
```

Comment:
why do we need the custom dirs again?
I think I'm starting to understand the need for the extra stanza on cmake configs.